source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+Mattermost provides a self-hosted Push Proxy you can deploy, the Mattermost Push Notification Service (MPNS). This is also available via a hosted-by-Mattermost option, or Hosted Push Notification Service (HPNS)*.
+
+    **Note:** `Only Mattermost Enterprise Edition E10 <https://about.mattermost.com/pricing/>`_ or higher to receive access to our Hosted Push Notification Service (HPNS)
+
+If you will be using the Mattermost applications via Google Play and Apple App Store, the HPNS is all you need.
+
+Our Hosted Push Notification Service offers:
+
+- Access to a publicly-hosted Mattermost Push Notification Service (MPNS).
+- An explicit `privacy policy <https://about.mattermost.com/hpns-privacy/>`_ for the contents of unencrypted messages.
+- Encrypted TLS connections:
+    - between HPNS and Apple Push Notification Services
+    - HPNS and Google’s Firebase Cloud Messaging service
+    - HPNS and your Mattermost server
+- Production-level uptime expectations
+- Compatibility with EMM Providers*
+
+\* *When using our publicly-available app store applications and the AppConfig standard.*
+
+Of course, if you would like to host your own version of the MPNS, that is always an option but requires a custom built app. The `Choosing the Right Deployment Model <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.6zz62c4mus3l>`_ section of this guide, as well as our `Mobile App Admin Documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ are the best places to start.
+
+********************************************
+Enterprise Mobile Management (EMM) Providers
+********************************************
+
+EMM Providers develop software that helps enterprise teams manage secure mobile technology deployments. This includes the use of mobile devices and usage-ready applications.
+
+Most large enterprise teams are familiar with `Enterprise Mobile Management <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`_ providers, or EMMs. If this is a new term for you, review `Appendix B <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.pn9oiqr3rebj>`_ for a list of providers and relevant information.
+
+For those taking on larger deployments, we assume you are already using an EMM provider.
+
+AppConfig is newer, more modern approach when compared to App Wrapping.
+
+We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
+
+While app wrapping has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
+
+AppConfig (Supported) vs. App Wrapping (Not Supported)
+======================================================
+
+Mattermost only supports the AppConfig standard. It does not support app wrapping. If you proceed with app wrapping, it will lead to an inability to use the mobile app.
```

Comment:
```suggestion
Mattermost only supports the AppConfig standard. It does not support app wrapping. If you proceed with app wrapping, it may lead to additional deployment time, and can limit overall success of your mobile deployment. 
```